src/Mods.jl

```diff
@@ -1,33 +1,33 @@
 module Mods
 
-import Base: isequal, (==), (+), (-), (*), (inv), (/), (//), (^), hash, show
-import Base: zero, one, rand, conj
```

Comment:
They are automatically generated. Since Mod subtypes Number, it will use `Mod{N,T}(0)` as the zero element and `Mod{N,T}(1)` as the one and oneunit. `iszero` can be defined by `== 0`. Since iszero is more efficient, I switch to defining `iszero` first and use `iszero` for defining `==` in the new commits.

Reply:
ok.